docs/integrations.md

```diff
@@ -1,21 +1,127 @@
 # Integrations
 
-> Note that all integrations are currently under development and are not officially supported. Please contact semgrep@r2c.dev or [join our slack](https://join.slack.com/t/r2c-community/shared_invite/enQtNjU0NDYzMjAwODY4LWE3NTg1MGNhYTAwMzk5ZGRhMjQ2MzVhNGJiZjI1ZWQ0NjQ2YWI4ZGY3OGViMGJjNzA4ODQ3MjEzOWExNjZlNTA) for help getting started
+This document describes integrating `semgrep` into other pieces of software.
 
-## Github Action
+Contents:
```

Comment:
Some high level meta comments:

Would it be useful to include 1-2 links to supporting docs for each CI platform? Makes it easier if the user wants to slightly adjust something, if something breaks, etc.

A couple of places there's `/path/to/code` - do CI platforms tend to mount code in the same location? If not, what's the default place for each platform? 
* Can we make this placeholder more explicit so there's no ambiguity about how to change it to get up and running? (e.g. `/root/code/<repo_name>/`)

Reply:
> Would it be useful to include 1-2 links to supporting docs for each CI platform? Makes it easier if the user wants to slightly adjust something, if something breaks, etc.

Yup, definitely, let's do it

> A couple of places there's /path/to/code - do CI platforms tend to mount code in the same location? If not, what's the default place for each platform?

I had a similar thought on this. I _think_ they tend to mount the code then change the working directory to it before starting your commands, so we could probably just use `.` (or nothing at all since that's what `semgrep` defaults to).